index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    
+    <link rel="stylesheet" href="style.css">
+
+    <title>Sanam's Portfolio</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    
+<section class="head">
+      <header class="header">
+        <nav class="nav-menu">
+            <a href="#"> <img src="images\Logo Placeholder.svg" alt="logo"></a>

```

Comment:
Kindly remove the logo image and placed it outside the `nav` tag but inside the `header` tag, the nav should only wrap around the nav links according to the HTML semantic usage. You can check this [documentation](https://www.w3schools.com/html/html5_semantic_elements.asp) for more information. 😉 

```
 <header class="header">
         <a href="#"> <img src="images\Logo Placeholder.svg" alt="logo"></a>
        <nav class="nav-menu">
      <div class="nav-menu">
                <ul class="menu-desktop" href="#">
                    <li><a href="my-work">Portfolio</a></li>
                    <li><a href="about-me">About</a></li>
                    <li><a href="my-work">Contact</a></li>
                </ul>
</nav>
```